third_party/txt/src/minikin/LineBreaker.cpp

```diff
@@ -159,22 +157,13 @@ float LineBreaker::addStyleRun(MinikinPaint* paint,
   size_t postSpaceCount = mSpaceCount;
   for (size_t i = start; i < end; i++) {
     uint16_t c = mTextBuf[i];
-    if (c == CHAR_TAB) {
-      mWidth = mPreBreak + mTabStops.nextTab(mWidth - mPreBreak);
-      if (mFirstTabIndex == INT_MAX) {
-        mFirstTabIndex = (int)i;
-      }
-      // fall back to greedy; other modes don't know how to deal with tabs
-      mStrategy = kBreakStrategy_Greedy;
-    } else {
-      if (isWordSpace(c))
-        mSpaceCount += 1;
-      mWidth += mCharWidths[i];
-      if (!isLineEndSpace(c)) {
-        postBreak = mWidth;
-        postSpaceCount = mSpaceCount;
-        afterWord = i + 1;
-      }
+    if (isWordSpace(c))
```

Comment:
Add a "libtxt:" comment here saying that tab handling has been removed.

(This is intended to keep track of how our version of Minikin differs from the original)

Reply:
Got it.  Thanks for the review.